install/kubernetes/helm/istio/values.yaml

```diff
@@ -96,6 +96,7 @@ certmanager:
 #
 istio_cni:
   enabled: false
+  istio_cni_network: "istio-cni"
```

Comment:
If this is a new field (i.e., we don't need to keep api), why not just call it `network`? It's already within a `istio_cni` structure.

Reply:
It's not a new field, it's already in use within the operator